### PR TITLE
Fix LaTeX file references (v2)

### LINKED ISSUE #411: LatexWriter cannot find images generated from examples
I have a few `@example` sections in the documentation, which generate a PNG using Gadfly. That PNG is then referenced in the doc in markdown: ``

This works fine for HTML documents. However, when using the LatexWriter, the images are not found during pdf generation. I believe this is due to the generation happening in a temporary directory : https://github.com/JuliaDocs/Documenter.jl/blob/3e5e4792e6de672df4378d7d91ee1f376fc66aff/src/Writers/LaTeXWriter.jl#L46

When I manually run `latexmk` in the build directory (copying documenter.sty), everything works. 

Not sure what is the best way to fix this?

### PR DESCRIPTION
**Pull request by mortenpi**
Based on #424. Sorry, @aviks, I noticed an issue with the paths and got sort of carried away. I've retained your original contribution, I just merged the commits, removed a space and made small edits to the commit messages. I hope it is fine that it gets merged this way.

Closes #424, fixes #411.

### CODE REVIEW

File: test/utilities.jl [hunk 1]
```diff
@@ -65,6 +65,13 @@ end
     @test OuterModule in Documenter.Utilities.submodules(OuterModule)
     @test OuterModule.InnerModule in Documenter.Utilities.submodules(OuterModule)
     @test length(Documenter.Utilities.submodules(OuterModule)) == 2
+
+    @test Documenter.Utilities.isabsurl("file.md") === false
+    @test Documenter.Utilities.isabsurl("../file.md") === false
+    @test Documenter.Utilities.isabsurl(".") === false
+    @test Documenter.Utilities.isabsurl("https://example.org/file.md") === true
+    @test Documenter.Utilities.isabsurl("http://example.org") === true
+    @test Documenter.Utilities.isabsurl("ftp://user:pw@example.org") === true
```
Comment (aviks) [inline test/utilities.jl]: Add a test for a filesystem absolute URL eg. `/home/me/myself.md` ??

**Comment by aviks** [comment]
Thanks Morten.

**Comment by aviks** [comment]
Bump. Anything we can do speed this along?

File: test/utilities.jl (hunk 1 above)
Comment (mortenpi) [inline test/utilities.jl]: That's actually a relative URL technically, but a test would be useful nevertheless.

The interpretation of such an URL should be that they're relative to the root of the document (and not relative to the filesystem root). I added a special case for that to the LaTeX code as well.

**Comment by mortenpi** [comment]
I am planning to tag a patch version this weekend with a couple of the bugfix PRs and I'll include this as well. Sorry for being a bit slow on this.

**Pull request merged**